### PR TITLE
Add KubeRay related blogs

### PR DESCRIPTION
**Pull request by tedhtchang**
## Why are these changes needed?

Adding KubeRay related articles

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ x] This PR is not tested :(

### CODE REVIEW

**Comment by kevin85421** [comment]
@gvspraveen would you mind reviewing this PR? Thanks!

File: README.md [hunk 1]
```diff
@@ -26,11 +26,16 @@ by some organizations to back user interfaces for KubeRay resource management.
...
 * [Nginx](docs/guidance/ingress.md)
-* [Prometheus and Grafana](docs/guidance/prometheus-grafana.md) 
+* [Prometheus and Grafana](docs/guidance/prometheus-grafana.md)
 * [Volcano](docs/guidance/volcano-integration.md)
 * [MCAD](docs/guidance/kuberay-with-MCAD.md)
 * [Kubeflow](docs/guidance/kubeflow-integration.md)
 
+## Blogs
+
+* [AI/ML Models Batch Training at Scale with Open Data Hub](https://cloud.redhat.com/blog/ai/ml-models-batch-training-at-scale-with-open-data-hub)
```
Comment (kevin85421) [inline README.md:36]: Maybe we can add the name of the company and the publication date?

**Review by kevin85421 — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (tedhtchang) [reply README.md]: @kevin85421 Sounds good. Added.

**Pull request merged**